launchpad_backend/launchpad_server/routes/startup_data.py

```diff
@@ -2,72 +2,749 @@
+    Design - Learns and contributes to processes for the architecture of a product/solution feature and learns to create proposals by testing design hypotheses and helping to refine code plans under the technical leadership of others. Produces code to test hypotheses for technical solutions and assists with technical validation efforts. Helps with and participates in the development of design documents for simple designs or user stories with oversight, helps to determine the technology that will be leveraged, and how they will interact.
+    Coding - With guidance, learns to create and implement code for a product, service, or feature reusing code as applicable. Writes and learns to create code that is extensible and maintainable. Learns about and applies diagnosability, reliability, and maintainability, and understands when the code is ready to be shared and delivered. Apply coding patterns and best practices to write code.
+    Implement - Learns to review work items to gain knowledge of product features in partnership with appropriate stakeholders (e.g., project managers). Assists and learns about breaking down work items into tasks and provides estimation.
+    Reliability and Supportability - Learns about and contributes to operations of live service as issues arise on a rotational, on-call basis. Identifies solutions and mitigations to simple issues impacting performance or functionality of Live Site services.
+    Engineering Excellence - Reviews current developments and proactively seeks new knowledge that will improve the availability, reliability, efficiency, observability, and performance of products while also driving consistency in monitoring and operations at scale.
+    Embody our culture and values.
+
+    Qualifications
+
+    Bachelor's Degree in Computer Science, or related technical discipline with proven experience coding in languages including, but not limited to, C, C++, C#, Java, JavaScript, or Python
+
+    Understanding of Computer Science fundamentals, data structures, algorithms, operating systems, design patterns and related topics
+
+    Microsoft is an equal opportunity employer. Consistent with applicable law, all qualified applicants will receive consideration for employment without regard to age, ancestry, citizenship, color, family or medical care leave, gender identity or expression, genetic information, immigration status, marital status, medical condition, national origin, physical or mental disability, political affiliation, protected veteran or military status, race, ethnicity, religion, sex (including pregnancy), sexual orientation, or any other characteristic protected by applicable local laws, regulations and ordinances. If you need assistance and/or a reasonable accommodation due to a disability during the application process, read more about requesting accommodations.""",
```

Comment:
Can you copy paste this string somewhere in the frontend and try displaying it. Im not sure but I have a feeling its not going to respect all the blank lines and indents. I think youll have to use "\n" for new lines and something else for indentation. Let me know though, maybe it works without